idd/Energy+.idd.in

```diff
@@ -21018,188 +21018,233 @@ ZoneContaminantSourceAndSink:Generic:DepositionRateSink,
 \group Daylighting
 
 Daylighting:Controls,
-       \min-fields 19
-       \memo Dimming of overhead electric lighting is determined from
-       \memo interior daylight illuminance calculated at one or two reference points.
-       \memo reference points are given in coordinates specified in the GlobalGeometryRules object
-       \memo Daylighting Reference Point CoordinateSystem field
+       \extensible:3 - repeat last three fields remembering to remove ; from "inner" fields.
+       \min-fields 16
+       \memo Dimming of overhead electric lighting is determined from each reference point.
        \memo Glare from daylighting is also calculated.
-  A1 , \field Zone Name
+  A1,  \field Name
+       \required-field
+       \type alpha
+  A2,  \field Zone Name
        \required-field
        \type object-list
        \object-list ZoneNames
-  N1 , \field Total Daylighting Reference Points
+  A3,  \field Daylighting Method
+       \required-field
```

Comment:
@JasonGlazer I would do the opposite, keep the default and ditch \required-field.  The default ensures that the field will always have a value even if the user leaves it blank.  Maybe it's a subtle difference, but \required-field to me says there *must* be a value in the idf for this field.